pkg/apimachinery/utils/meta.go

```diff
@@ -34,6 +34,9 @@ const AnnoKeyRepoPath = "grafana.app/repoPath"
 const AnnoKeyRepoHash = "grafana.app/repoHash"
 const AnnoKeyRepoTimestamp = "grafana.app/repoTimestamp"
 
+// LabelKeyDeprecatedInternalID will be removed in Grafana 13. Do not add any new usage of it.
```

Comment:
```suggestion
// Deprecated: LabelKeyDeprecatedInternalID will be removed in Grafana 13. Do not add any new usage of it.
```

Reply:
making these deprecated for real will require everything to have explicit lint exceptions to use them